base/system/format/format.c

```diff
@@ -436,6 +437,7 @@ int wmain(int argc, WCHAR *argv[])
 
         case DRIVE_NO_ROOT_DIR:
             K32LoadStringW(GetModuleHandle(NULL), STRING_NO_VOLUME, szMsg, ARRAYSIZE(szMsg));
+            // FIXME, GetLastError(): K32LoadStringW() can set it, GetDriveTypeW() does not...
             PrintWin32Error(szMsg, GetLastError());
```

Comment:
```suggestion
            ConResPuts(StdOut, STRING_NO_VOLUME);
```
And it would be ideal to update these strings in resources too (ie. end with a dot and null-terminate if necessary).

Reply:
Adapted.